Data/Uniques/bow.lua

```diff
@@ -38,6 +38,44 @@ Adds 25 to 50 Fire Damage
 {variant:4}50% More Bow Damage at Close Range
 Bow Knockback at Close Range
 ]],[[
+The Crimson Storm
+Steelwood Bow
+Variant: Crit Multi while Rare/Unique Nearby
+Variant: Attack Speed while Rare/Unique Nearby
+Variant: Damage per Frenzy Charge
+Variant: Damage per Power Charge
+Variant: Damage per Endurance Charge
+Variant: Accuracy and Quality
+Variant: Attack Speed and Quality
+Variant: Attack Speed/Trigger Blood Rage on Kill
+Variant: Minion Attack and Cast Speed
+Variant: Double Damage
+Variant: Double Damage while Foccused
+Variant: Socketed Spell Trigger
+Requires Level 57, 190 Dex
+(140–170)% increased Physical Damage
+(25–35)% increased Critical Strike Chance
+50% chance to inflict Bleeding on Critical Strike with Attacks
+Enemies you inflict Bleeding on grant (60–100)% increased Flask Charges
+Adds (100–120) to (150–165) Physical Damage against Bleeding Enemies
+50% chance to Maim Enemies on Critical Strike with Attacks
+{variant:1}{crafted}+(18-45)% Critical Strike Multiplier while there is a Rare or Unique Enemy Nearby
+{variant:2}{crafted}(11-22)% increased Attack Speed while a Rare or Unique Enemy is Nearby
+{variant:3}{crafted}(5-6)% increased Damage per Frenzy Charge
+{variant:4}{crafted}(5-6)% increased Damage per Power Charge
+{variant:5}{crafted}(5-6)% increased Damage per Endurance Charge
+{variant:6}{crafted}+(51-250) to Accuracy Rating
+{variant:6}{crafted}+(10-18)% to Quality
```

Comment:
```suggestion
{variant:6}{crafted}+(7-18)% to Quality
```
Can have lower minimal value (same as Attack Speed/Quality variant)